pycode/memilio-epidata/memilio/epidata_test/test_epidata_get_jh_data.py

```diff
@@ -189,6 +306,112 @@ def test_get_JH_Data_Download(self, mock_file):
         self.assertEqual(df[(df["CountryRegion"] == 'France') & (df["Date"] == "2020-09-26") & (
             df['ProvinceState'] == 'Martinique')]["Deaths"].item(), 20)
 
+    @patch('memilio.epidata.getJHData.gd.get_file')
+    def test_get_JH_Data_Download_omit_raw(self, mock_loadcsv):
```

Comment:
@reneSchm is there a reason why you undid this change? I removed the unittest because it tests the download of the JH Data.

Reply:
Do we not want to test this data, or is this test redundant?
I assumed the test would still be relevant, as it was updated on main. If it is not, we can delete it again.